tutorials/notebooks/example_pytorch_ssdlite_mobilenetv3.ipynb

```diff
@@ -0,0 +1,459 @@
+    "\n",
+    "        batch_detections = format_results(_outs, img_ids)  # Implement this function\n",
+    "\n",
+    "        self.all_detections.extend(batch_detections)\n",
+    "\n",
+    "    def result(self):\n",
+    "        # Initialize COCO evaluation object\n",
+    "        self.coco_dt = self.coco_gt.loadRes(self.all_detections)\n",
+    "        coco_eval = COCOeval(self.coco_gt, self.coco_dt, 'bbox')\n",
+    "\n",
+    "        # Run evaluation\n",
+    "        coco_eval.evaluate()\n",
+    "        coco_eval.accumulate()\n",
+    "        coco_eval.summarize()\n",
+    "\n",
+    "        # Print mAP results\n",
+    "        print(\"mAP: {:.4f}\".format(coco_eval.stats[0]))\n",
+    "\n",
+    "        return coco_eval.stats\n",
+    "\n",
+    "    def reset(self):\n",
+    "        self.all_detections = []"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3cde2f8e-0642-4374-a1f4-df2775fe7767",
+   "metadata": {},
+   "source": [
+    "#### Evaluate float model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56393342-cecf-4f64-b9ca-2f515c765942",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "EVAL_DATASET_FOLDER = '/path/to/coco/evaluation/images/val2017'\n",
+    "EVAL_DATASET_ANNOTATION_FILE = '/path/to/coco/annotations/instances_val2017.json'\n",
+    "\n",
+    "\n",
+    "def collate_fn(batch_input):\n",
+    "    images = [b[0] for b in batch_input]\n",
+    "    targets = [b[1] for b in batch_input]\n",
+    "    return images, targets\n",
+    "\n",
+    "\n",
+    "coco_eval = torchvision.datasets.CocoDetection(root=EVAL_DATASET_FOLDER,\n",
+    "                                               annFile=EVAL_DATASET_ANNOTATION_FILE,\n",
+    "                                               transform=torchvision.transforms.ToTensor())\n",
+    "batch_size = 50\n",
+    "data_loader = torch.utils.data.DataLoader(coco_eval, batch_size=batch_size, shuffle=False,\n",
+    "                                          num_workers=0, collate_fn=collate_fn)\n",
+    "\n",
+    "coco_metric = CocoEval(EVAL_DATASET_ANNOTATION_FILE)\n",
+    "for batch_idx, (images, targets) in enumerate(data_loader):\n",
+    "    # Run inference on the batch\n",
+    "    images = list(image.to(device) for image in images)\n",
+    "    with torch.no_grad():\n",
+    "        outputs = model(images)\n",
+    "\n",
+    "    coco_metric.add_batch_detections(outputs, targets)\n",
+    "    if (batch_idx+1) % 10 == 0:\n",
+    "        print(f'processed {(batch_idx+1)*data_loader.batch_size} images')\n",
+    "\n",
+    "# Print mAP results\n",
+    "print(\"mAP: {:.4f}\".format(coco_metric.result()[0]))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "015e760b-6555-45b4-aaf9-500e974c1d86",
+   "metadata": {},
+   "source": [
+    "### Extract model to be quantized\n",
+    "\n",
+    "Extract the float model's backcone and head, and construct a torch model the only runs them"
```

Comment:
"Construct a torch model that exclusively contains only the backcone and head of the float model."